libs/store/data/src/lib/root.ts

```diff
@@ -42,6 +51,7 @@ export const entitiesReducer = combineReducers({
   projects: projects,
   project: projectReducer,
   environments: environments,
```

Comment:
Can we group projects/project and environments/environment inside the same keys, like :

```
project: {
   projects
   deploymentRules
}
```

and 

```
environment: {
   environmentDeploymentRules
   environments
}
```

?